app/views/projects/show.html.erb

```diff
@@ -12,8 +12,25 @@
       .html_safe
   %>;
 
+  var taxon_lookup = <%=
```

Comment:
I'm not sure that this view-specific code can move to a module, but all the jQuery below needs to be refactored into one for sure.